code/_onclick/telekinesis.dm

```diff
@@ -121,16 +121,15 @@ var/const/tk_maxrange = 15
 		focus_object(target, user)
 		return
 
-	if(focus.anchored || !isturf(focus.loc))
+	if(focus.anchored)
 		qdel(src)
-		return
 
 	if(target == focus)
 		target.attack_self_tk(user)
 		return // todo: something like attack_self not laden with assumptions inherent to attack_self
 
 
-	if(!isturf(target) && istype(focus,/obj/item) && target.Adjacent(focus))
+	if(!istype(target, /turf) && istype(focus,/obj/item) && target.Adjacent(focus))
```

Comment:
changing isturf to istype(x, /turf) is unnecessary, really. isturf is actually cleaner, keep it.